Patches/Kurin/ThingDef_Misc/DRNTF_Apparel.xml

```diff
@@ -1,236 +1,321 @@
+				defName="Kurin_OnSkin_Open_Shoulder_Knit" or
+				defName="Kurin_OnSkin_Hoodie" or
+				defName="Kurin_OnSkin_Casual_Wear"
+				]/statBases/StuffEffectMultiplierArmor</xpath>
+				<value>
+					<StuffEffectMultiplierArmor>1</StuffEffectMultiplierArmor>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Kurin_OnSkin_Gothic_Dress" or 
+				defName="Kurin_OnSkin_Nun_Dress" or
+				defName="Kurin_OnSkin_Work_Jumpsuit" or
+				defName="Kurin_OnSkin_Traditional_Simple_Dress" or
+				defName="Kurin_OnSkin_Traditional_Fancy_Dress" or
+				defName="Kurin_OnSkin_Traditional_Luxury_Dress" or
+				defName="Kurin_OnSkin_Cute_Dress" or
+				defName="Kurin_OnSkin_Seductive_White_Dress" or
+				defName="Kurin_OnSkin_Seductive_Black_Dress"
+				]/statBases/StuffEffectMultiplierArmor</xpath>
+				<value>
+					<StuffEffectMultiplierArmor>1.5</StuffEffectMultiplierArmor>
+				</value>
+			</li>
+			
+			<!--========= Onskin Armor =========-->
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Kurin_OnSkin_Armored_Gothic_Dress" or
+				defName="Kurin_OnSkin_Military_Uniform"
+				]/statBases/StuffEffectMultiplierArmor</xpath>
+				<value>
+					<StuffEffectMultiplierArmor>2.5</StuffEffectMultiplierArmor>
+					<Bulk>12</Bulk>
+					<WornBulk>6</WornBulk>
+					<Mass>6</Mass>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Kurin_OnSkin_Armored_Gothic_Dress"]/equippedStatOffsets/MeleeDodgeChance</xpath>
+				<value>
+					<MeleeDodgeChance>0.2</MeleeDodgeChance>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Kurin_OnSkin_Military_Uniform"]/equippedStatOffsets/ShootingAccuracyPawn</xpath>
+				<value>
+					<ShootingAccuracyPawn>0.4</ShootingAccuracyPawn>
+					<AimingAccuracy>0.2</AimingAccuracy>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Kurin_OnSkin_Armored_Gothic_Dress"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>1.25</ArmorRating_Sharp>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Kurin_OnSkin_Armored_Gothic_Dress"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>1.55</ArmorRating_Blunt>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Kurin_OnSkin_Military_Uniform"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>2.25</ArmorRating_Sharp>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Kurin_OnSkin_Military_Uniform"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>3.05</ArmorRating_Blunt>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Kurin_OnSkin_Bikini_Armor"
```

Comment:
This seems to be some kind of power armor or something, based on the bulk? Does need hands/feet patched to be covered as well?

Reply:
Not exactly powered armor, but it is based off of plate armor which also does provide hand and feet protection so I added. I updated the bulk/worn bulk to match.

Added the hand and feet to the military uniform as well, since it's pretty much meant to be an equivalent in terms of effectiveness/cost (slightly less bulky but slighly less protective when made of hyperweave).